### DISCUSSION TITLE
UI/UX: The addon manager needs some work/love for Freecad 1.0

### DISCUSSION THREAD

**Discussion by leoheck** [FreeCAD/FreeCAD#17163] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

There is something off with this Addon Manager since its first day.
The addon manages is a UI/UX mess.

Information is always missing when freecad is launched. It has to fetch everything again every time the window opens.
See the image here, it knows the package names, but it does now nothing about the description... 

![image](https://github.com/user-attachments/assets/6f06359d-f704-4e8c-932f-2a23a1190353)

Updates should be carried on the backstage... the current information should always be displayed. 

=================================

Also these 2 buttons don't work.

If there are updates, this first button is enabled but I cannot see the stuff that need to be updated without having to update them. Also, the Python dependencies do not work. 

![image](https://github.com/user-attachments/assets/ebe02ef2-2699-4d2d-be9d-5f19f2ecfcbb)

=================================

Also, this button here updates all python packages. there is no way to update just a subset of things.
Also, this is not a button, it looks like a scrool bar.. it increases forever with the screen.

![image](https://github.com/user-attachments/assets/97ef325e-3def-468c-a401-4cfd23a9e55f)

### Full version info

```shell
OS: Ubuntu 24.04.1 LTS (ubuntu:GNOME/ubuntu-xorg)
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38923 (Git) AppImage
Build type: Release
Branch: main
Hash: d20cb9e6ee198beb2bfd7e72d3dec0a575e3f28c
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)
Stylesheet/Theme/QtStyle: OpenDark.qss/OpenDark/Qt default
Installed mods: 
  * Assembly4 0.50.16-dev
  * sheetmetal 0.4.26
  * InventorLoader 1.4.0
  * diff-tools 0.1.0
  * ose-piping
  * OpenTheme 2024.9.1
  * fasteners 0.5.29
  * osh-autodoc-workbench 0.2.3
  * OpticsWorkbench 1.0.24
  * lattice2 1.0.0
  * kicadStepUpMod 11.3.9
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bofdahof** [comment]
Was going to have a look at this after my current batch of issue reports as there are certainly issues in addonmanager, but find it flicked over here. How does that work?
